ui/render/denoiser.py

```diff
@@ -38,22 +39,23 @@ def draw(self, context):
                                 sub, "Running denoiser...")
 
         col = layout.column(align=True)
-        col.prop(denoiser, "type", expand=False)
+        #col.prop(denoiser, "type", expand=False)
         col.enabled = denoiser.enabled and not LuxCoreRenderEngine.final_running
 
-        if denoiser.enabled and denoiser.type == "BCD":
-            if config.get_sampler() == "METROPOLIS" and not config.use_tiles:
-                layout.label(text="Metropolis sampler can lead to artifacts!", icon=icons.WARNING)
+        #if denoiser.enabled and denoiser.type == "BCD":
+            #if config.get_sampler() == "METROPOLIS" and not config.use_tiles:
+                #layout.label(text="Metropolis sampler can lead to artifacts!", icon=icons.WARNING)
 
-        if denoiser.type == "BCD":
-            sub = layout.column(align=True)
+        #if denoiser.type == "BCD":
+            #sub = layout.column(align=True)
             # The user should be able to adjust settings even when denoiser is disabled            
-            sub.prop(denoiser, "filter_spikes")
-            sub = layout.column(align=True)
-            sub.prop(denoiser, "hist_dist_thresh")
-            sub = layout.column(align=True)
-            sub.prop(denoiser, "search_window_radius")
-        elif denoiser.type == "OIDN":
+            #sub.prop(denoiser, "filter_spikes")
+            #sub = layout.column(align=True)
+            #sub.prop(denoiser, "hist_dist_thresh")
+            #sub = layout.column(align=True)
+            #sub.prop(denoiser, "search_window_radius")
+        #elif denoiser.type == "OIDN":
```

Comment:
There's a _lot_ of commented-out code in this PR. If you're intending to remove it, delete the lines. The original lines will still be available in the commit history.